MLAPI.Relay/MLAPI.Relay.csproj

```diff
@@ -46,4 +47,10 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
UNETServerDLL.so and UNETServerDLL.dylib also has to be included for it to work on macOS and Linux.

Reply:
Done :)